### PR TITLE
magento/devdocs#7375: GraphQL: Add support for Inventory In-Store Pickup.

### PR DESCRIPTION
**Pull request by swnsma**
Apply changes to devdocs dedicated to In-Store Pickup GraphQl support.

## Purpose of this pull request
https://github.com/magento/devdocs/issues/7375

## Affected DevDocs pages
- `src/guides/v2.4/graphql/queries/cart.md`
- `src/guides/v2.4/graphql/mutations/set-shipping-address.md`
- `src/_data/toc/graphql.yml`
- `src/guides/v2.4/graphql/queries/pickup-locations.md`

whatsnew
Added the [`pickupLocations` query](https://devdocs/magento.com/guides/v2.4/graphql/queries/pickup-locations.html), which returns a list of locations that customers can use to pick up their orders.

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

**Comment by swnsma** [comment]
Hello @rogyar 
Schema adjustments are here:
- https://github.com/magento/inventory/blob/1.2.0-develop/InventoryInStorePickupGraphQl/etc/schema.graphqls
- https://github.com/magento/inventory/blob/1.2.0-develop/InventoryInStorePickupQuoteGraphQl/etc/schema.graphqls

File: src/guides/v2.4/graphql/queries/pickup-locations.md [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+`pageSize` | Int | The maximum number of pickup locations to return at once. The attribute is optional. The default value is 20.
+`currentPage` | Int | Specifies which page of results to return. The default value is 1.
+
+### AreaInput object {#AreaInput}
+
+Use the `AreaInput` to apply filtration by distance. All attributes are required.
+
+Attribute | Data type | Description
+--- | --- | ---
+`radius` | Int | The radius for the search in KM.
```
Comment (rogyar) [inline src/guides/v2.4/graphql/queries/pickup-locations.md:123]: If the attribute is mandatory, please, highlight it in the "Data type" column
```suggestion
`radius` | Int! | The radius for the search in KM.
```

It applies for all other required attributes

File: src/guides/v2.4/graphql/queries/pickup-locations.md [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+`city` | String | City where pickup location is placed.
+`street` | String | Street where pickup location is placed.
+`postcode` | String | Postcode where pickup location is placed.
+`phone` | String | Contact phone number of the pickup location.
+
+### SearchResultPageInfo object {#SearchResultPageInfo}
+
+Attribute | Data type | Description
+--- | --- | ---
+current_page | Int | Specifies which page of results to return
```
Comment (rogyar) [inline src/guides/v2.4/graphql/queries/pickup-locations.md:201]: Please, use the same formatting for the attributes name
```suggestion
`current_page` | Int | Specifies which page of results to return
```

**Comment by swnsma** [comment]
@rogyar, done!
Please re-check!

**Comment by keharper** [comment]
running tests

**Comment by keharper** [comment]
running tests

**Pull request merged**